docs/testing/tips.rst

```diff
@@ -0,0 +1,10 @@
+Notes for Cypress testing
+=========================
+
+Https not Https
+---------------
+
+When tests are run within GitHub's environments, calls to external urls must be https, otherwise they will not run. 
+This applies even to fixtures that don't even make external requests. 
+
+
```

Comment:
I've created this as a stub to record my testing discoveries. We will need to create more documentation about testing, but this felt important enough to kickstart the process.